controller/proxy-injector/webhook_test.go

```diff
@@ -103,17 +111,18 @@ func TestIgnore(t *testing.T) {
 			t.Fatal("Unexpected error: ", err)
 		}
 
-		if !webhook.ignore(deployment) {
+		ignore, err := webhook.ignore(deployment)
+		if err != nil {
+			t.Fatalf("Unexpected ignore error: %s", err)
+		}
+		if !ignore {
```

Comment:
`t.Fatalf` aborts immediately, whereas `t.Errorf` fails the test but allows it to continue executing. If the `webhook.ignore()` call returned an error, we should abort immediately, since the value of `ignore` is meaningless at that point.

I tend to use `t.Fatalf` everywhere, since I don't see a lot of value in continuing with a test that's going to fail. I'll update this test accordingly.